Version 2 (#73)
YunoHost-Apps/lutim_ynh: doc/ADMIN.md

> ## Configuration
> 
> Edit the file `__INSTALL_DIR__/lutim.conf` to configure Lutim.
> Then reload the lutim service.

Reviewer: ```suggestion
You can edit `__INSTALL_DIR__/lutim.conf` to configure Lutim, then reload the `lutim` service.
```
